core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/validation/ValidateRequest.java

```diff
@@ -102,13 +107,17 @@ private ValidationResult doValidate(BranchContext context, Writer index) throws
 		final List<Promise<Object>> validationPromises = Lists.newArrayList();
 		// evaluate selected rules
 		for (ValidationRule rule : rules) {
+			RemoteJob remoteJob = context.service(RemoteJob.class);
```

Comment:
This might result in a missing service exception due to the fact that Request implementations are not always wrapped and run in jobs. Please remove this line.